packages/chakra-ui/src/AltDateWidget/AltDateWidget.tsx

```diff
@@ -11,12 +11,19 @@ import {
 } from '@rjsf/utils';
 import { getChakra } from '../utils';
 
-function AltDateWidget<T = any, S extends StrictRJSFSchema = RJSFSchema, F extends FormContextType = any>(
-  props: WidgetProps<T, S, F>,
-) {
-  const { autofocus, disabled, id, onBlur, onFocus, options, readonly, registry } = props;
+function AltDateWidget<T = any, S extends StrictRJSFSchema = RJSFSchema, F extends FormContextType = any>({
+  autofocus = false,
+  disabled = false,
+  readonly = false,
+  time = false,
+  options = {
+    yearsRange: [1900, new Date().getFullYear() + 2],
+  },
+  ...props
+}: WidgetProps<T, S, F>) {
+  const { id, onBlur, onFocus, registry } = props;
   const { translateString } = registry;
-  const { elements, handleChange, handleClear, handleSetNow } = useAltDateWidgetProps(props);
+  const { elements, handleChange, handleClear, handleSetNow } = useAltDateWidgetProps({ ...props, options });
```

Comment:
This code is equivalent to what the code in `getWidget()` does.  You will want to remove the default assignment above
```suggestion
  const realOptions = { yearsRange: [1900, new Date().getFullYear() + 2], ...options };
  const { elements, handleChange, handleClear, handleSetNow } = useAltDateWidgetProps({ ...props, realOptions });
```